pkgs/ffigen/test/code_generator_tests/code_generator_test.dart

```diff
@@ -361,6 +386,8 @@ void main() {
                   NativeType(
                     SupportedNativeType.Int8,
                   ),
+                  // Arrays are always supported in struct fields
+                  useArrayType: true,
```

Comment:
Just a note: Struct/Union (compound.dart) handle generation of ConstantArray members in their own file, so this flag will probably have no effect here.

Reply:
I've clarified the comment.